erpnext/templates/pages/order.py

```diff
@@ -24,6 +24,12 @@ def get_context(context):
 
 	context.enabled_checkout = frappe.get_doc("Shopping Cart Settings").enable_checkout
 
+	default_print_format = frappe.db.sql("""select value from `tabProperty Setter` where property='default_print_format' and doc_type='{0}'""".format(frappe.form_dict.doctype), as_dict=True)
```

Comment:
`frappe.db.get_value('Property Setter', dict(property='default_print_format', doc_type=frappe.form_dict.doctype)`

Reply:
Thanks @rmehta. I have corrected it!